src/Compilers/Core/Portable/Operations/OperationNodes.cs

```diff
@@ -26,7 +23,23 @@ internal sealed class NoneOperation : Operation
             OperationConstantValue = constantValue;
         }
 
-        public override IEnumerable<IOperation> Children { get; }
+        internal ImmutableArray<IOperation> Children { get; }
+
+        protected override IOperation GetCurrent(int slot, int index)
+            => slot switch
+            {
+                0 when index < Children.Length => Children[index],
```

Comment:
>0 when index < Children.Length => Children[index], [](start = 16, length = 50)

IMHO. Consider either moving the ```=> ...``` part on a new line, or using a regular switch statement.

Reply:
Similar suggestion applies to other similar changes in this file.

---
In reply to: [539594643](https://github.com/dotnet/roslyn/pull/49804#discussion_r539594643) [](ancestors = 539594643)